tests/integration/backend_test.py

```diff
@@ -762,50 +765,81 @@ def test_device_state(
 
 @pytest.mark.skipif(skip_remote_tests, reason=REASON)
 @pytest.mark.parametrize(
-    "authenticated_quum_backend", [{"device_name": "H1-1SC"}], indirect=True
+    "authenticated_quum_backend_qa", [{"device_name": "H1-1SC"}], indirect=True
+)
+@pytest.mark.parametrize(
+    "language",
+    [
+        Language.QASM,
+        Language.QIR,
+    ],
+)
+@pytest.mark.timeout(120)
+def test_wasm_qa(
+    authenticated_quum_backend_qa: QuantinuumBackend, language: Language
+) -> None:
+    wasfile = WasmFileHandler(str(Path(__file__).parent.parent / "wasm" / "add1.wasm"))
+    c = Circuit(1)
+    c.name = "test_wasm"
+    a = c.add_c_register("a", 8)
+    c.add_wasm_to_reg("add_one", wasfile, [a], [a])
+
+    b = authenticated_quum_backend_qa
+
+    c = b.get_compiled_circuit(c)
+    h = b.process_circuits(
+        [c], n_shots=10, wasm_file_handler=wasfile, language=language  # type: ignore
+    )[0]
+
+    b.get_result(h)
```

Comment:
Can we assert something useful about the result; it doesn't seem like a proper test otherwise.

Reply:
I have added a few lines, but I am not sure how helpful they are. See [7f52d74](https://github.com/CQCL/pytket-quantinuum/pull/357/commits/7f52d746a3157e4b689c96d4d1c21c2d4b7c22db)

This three tests are only submitting to the SC, they are only testing if this type of circuit is accepted, the result is always 0.